packages/webpack/src/config/base.js

```diff
@@ -189,10 +189,13 @@ export default class WebpackBaseConfig {
           // item in transpile can be string or regex object
           return !this.modulesToTranspile.some(module => module.test(file))
         },
-        use: perfLoader.js().concat({
-          loader: require.resolve('babel-loader'),
-          options: this.getBabelOptions()
-        })
+        use: perfLoader.js().concat(
+          {
+            loader: require.resolve('babel-loader'),
+            options: this.getBabelOptions()
+          },
+          'vue-jsx-hot-loader'
```

Comment:
Is that loader necessary? Nuxt is supporting JSX already (ref: https://github.com/nuxt/nuxt.js/blob/52a92fbc48510c7fbdaf3506345c0b686ee07b1f/examples/jsx/pages/about.js#L17)

Reply:
That one is for HMR when using jsx or tsx.